### PR TITLE
Refactoring loadPolicyLine

### PR DESCRIPTION
**Pull request by nodece**

### CODE REVIEW

File: src/adapter.ts [hunk 1]
```diff
@@ -56,12 +56,16 @@ export class SequelizeAdapter implements Adapter {
...
+                url: this.connStr,
+                logging: false,
+                pool: {max: 5, min: 0, idle: 10000}
+            };
             this.sequelize = new Sequelize(uriConfig);
         } else {
             await this.createDatabase();
             const url = this.connStr + 'casbin';
-            const uriConfig: ISequelizeUriConfig = {url, logging: false, pool: { max: 5, min: 0, idle: 10000 }};
+            const uriConfig: ISequelizeUriConfig = {url, logging: false, pool: {max: 5, min: 0, idle: 10000}};
```
Comment (hsluoyz) [inline src/adapter.ts]: Why is the above code returned but not this?

File: test/adapter.test.ts [hunk 2]
```diff
@@ -23,43 +23,41 @@ function testGetPolicy(e: Enforcer, res: string[][]) {
...
-    e = await Enforcer.newEnforcer('examples/rbac_model.conf', a);
-    testGetPolicy(e, [
-        ['alice', 'data1', 'read'],
-        ['bob', 'data2', 'write'],
-        ['data2_admin', 'data2', 'read'],
-        ['data2_admin', 'data2', 'write']]);
-
-    await a.close();
+    const a = await SequelizeAdapter.newAdapter('mysql://root:@localhost:3306/');
+    try {
```
Comment (hsluoyz) [inline test/adapter.test.ts:27]: Please do not use ``try``, just call ``a.close();`` at the end.

**Review by hsluoyz — COMMENTED** [review]
Please fix.

File: test/adapter.test.ts (hunk 2 above)
Comment (nodece) [reply test/adapter.test.ts:27]: When test is fail, db not close, jest not exit

File: src/adapter.ts (hunk 1 above)
Comment (nodece) [reply src/adapter.ts]: It's just that the code style has changed

**Pull request merged**